pkg/endpoint/log.go

```diff
@@ -162,7 +162,7 @@ func (e *Endpoint) updatePolicyLogger(fields map[string]interface{}) {
 		return
 	}
 
-	if !e.Options.IsEnabled(option.DebugPolicy) {
+	if !true {
```

Comment:
This seems suspect?

Reply:
Debug commit to figure out why CI is failing :see_no_evil: